### ISSUE TITLE
Display y value in bar charts

### ISSUE THREAD

**Issue by adnan449** [mcastorina/graph-cli#49]
How can I display the actual y value in the picture for each item?  For example, the value for C++, Nim etc?
![out](https://user-images.githubusercontent.com/70180150/114193842-41042e00-9992-11eb-8256-df8498d8f80a.png)

**Comment by mcastorina** [comment]
Currently that's not supported. A workaround would be to use `--text` and manually label each bar with its value, however I haven't tested text annotation on bar charts so it might be buggy.

**Comment by mcastorina** [comment]
Note to self when I implement this feature: https://stackoverflow.com/questions/28931224/adding-value-labels-on-a-matplotlib-bar-chart

**Comment by mcastorina** [comment]
`--bar-label` and `--bar-format FORMAT` flags have been added in ad9b0115dc28c21bc0c865b63b2ad2dff40a002e

```
graph samples/bar.csv --barh --bar-label --bar-format '$%.2f' --figsize 1600x1000 -o example.png
```
![example](https://user-images.githubusercontent.com/5395814/135958154-3b86ccdc-6fb8-4b5e-9cea-9b24fae8f9cf.png)

**Comment by mcastorina** [comment]
Now available in `v0.1.17`

Note that it depends on a `matplotlib` feature that was introduced in `v3.4.2`
